.git-authors

```diff
@@ -13,3 +14,4 @@ email:
 email_addresses:
    kj: karthijey@yahoo.com
    km: klausmana@outlook.com
+   nw: nathan.wang2@gmail.com
```

Comment:
Gmail is used by default if you see the `email` key, so you don't have to add an entry to `email_addresses`.

Reply:
So should I just include the entry as "nw: nathan.wang2" or not even include an email?

Reply:
Please remove '@gmail.com', only use 'nw: nathan.wang2'.